flink-connectors/flink-connector-elasticsearch-base/src/test/java/org/apache/flink/streaming/connectors/elasticsearch/table/IndexGeneratorFactoryTest.java

```diff
@@ -193,12 +202,26 @@ public void testDynamicIndexFromSystemTime() {
     }
 
     @Test
-    public void testDynamicIndexDefaultFormatTimestampWithLocalTimeZone() {
+    public void testDynamicIndexDefaultFormatTimestampWithLocalTimeZoneUTC() {
+        assumeThat(ZoneId.systemDefault(), is(ZoneId.of("UTC")));
```

Comment:
Why not use the same scheme with `assumingThat` as here: https://github.com/apache/flink/pull/19302/files#diff-718dd4539a392ce4b768bdfcabeca730bcdb4ffa4d09f691fd14c09002a84b95R124 ?

Reply:
The tests under `...streaming.connectors.elasticsearch` (for the old SinkFunction based connector) are still using Junit 4. I could port them but not sure if we should do that with tests that we want to remove soon anyway